README.md

```diff
@@ -1,5 +1,6 @@
-OS-X & Ubuntu: [![Build Status](https://travis-ci.org/dvidelabs/flatcc.svg?branch=master)](https://travis-ci.org/dvidelabs/flatcc)
+Ubuntu, macOS and Windows: [![Build Status](https://github.com/dvidelabs/flatcc/actions/workflows/ci.yml/badge.svg)](https://github.com/dvidelabs/flatcc/actions/workflows/ci.yml)
 Windows: [![Windows Build Status](https://ci.appveyor.com/api/projects/status/github/dvidelabs/flatcc?branch=master&svg=true)](https://ci.appveyor.com/project/dvidelabs/flatcc)
+[![Build Status](https://github.com/dvidelabs/flatcc/actions/workflows/weekly.yml/badge.svg)](https://github.com/dvidelabs/flatcc/actions/workflows/weekly.yml)
```

Comment:
Missing a description for the weekly build I think.

Reply:
Here is an example how the image looks:
![image text](https://github.com/Nordix/flatcc/actions/workflows/weekly.yml/badge.svg)
I skipped a description since the text Weekly is included in the image, but maybe there is a need for a description to get some distance to the windows badge.

Reply:
Well, I cannot see how it looks yet so I am just raising a potential concern, but I think it makes sense that all three builds have some text since otherwise it might see as if it is related to the Windows build, I guess. You could call extended build or something.